src/app/cli/src/full_test.ml

```diff
@@ -125,6 +117,10 @@ let run_test () : unit Deferred.t =
         `With_public_key
           (Public_key.compress largest_account_keypair.public_key)
       in
+      let fee = Currency.Fee.of_int in
+      let snark_work_fee, transaction_fee =
+        if proof_level = "full" then (fee 0, fee 0) else (fee 1, fee 2)
```

Comment:
Can’t this conditional be at the top level and not use inject?

Just noticed this is a test, so seems fine, tho I think it would be a bit cleaner